pkg/compare/compare.go

```diff
@@ -624,3 +719,20 @@ func (o Output) Print(format string, out io.Writer) (int, error) {
 	}
 	return n, nil
 }
+
+func (o Output) PrintPatches(out io.Writer) (int, error) {
```

Comment:
i think its better to create a separate object for the patches, where has both the load function and the print functions. no need to use the ouptput that contains final output of the tool or the diff sum. 

also because the patch file extends the api of the command, i think its better that is format is located in one   place. 
the code is a little to much divided between multiple places, that can make it hard to change and its harder to review.  

so i suggest on creating a struct called PatchFile that has list/map of the patchsave object (that can be just patch)
the patch file object can contain the print function (i would call it export instead or save), and also a load function. even put all of the patch functions in a different file